packages/@aws-cdk/aws-stepfunctions-tasks/lib/databrew/start-job-run.ts

```diff
@@ -0,0 +1,63 @@
+import * as iam from '@aws-cdk/aws-iam';
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import { Construct } from 'constructs';
+import { integrationResourceArn, validatePatternSupported } from '../private/task-utils';
+
+/**
+ * Properties for starting a job run with StartJobRun
+ */
+export interface DataBrewStartJobRunProps extends sfn.TaskStateBaseProps {
+
+  /**
+   * DataBrew Job to run
+   */
+  readonly name: string;
```

Comment:
clarification: does this mean that only jobs within the same account may be started? 
i.e. an ARN is not possible (which could reference a job in another account)

Reply:
Correct.

Reply:
gotcha thanks for clarifying